src/components/ReactP5Wrapper.tsx

```diff
@@ -3,4 +3,4 @@ import { memo } from "react";
 import { propsAreEqual } from "../utils/propsAreEqual";
 import { ReactP5WrapperGuard } from "./ReactP5WrapperGuard";
 
-export const ReactP5Wrapper = memo(ReactP5WrapperGuard, propsAreEqual);
+export const ReactP5Wrapper = memo(ReactP5WrapperGuard, propsAreEqual as any);
```

Comment:
We never use `as` without an unavoidable issue such as the one outlined in #207.